### ISSUE TITLE
Add JavaDocs to the library

### ISSUE THREAD

**Issue by minborg** [OpenHFT/Chronicle-Bytes#140]

**Comment by kailash-katharine** [comment]
Please clarify how I can help here. Thank you :)

**Comment by minborg** [comment]
So, the library does not have standard JavaDocs for most methods. I started to refactor core including documenting it but there were other issues with higher priority. 

I think a good start would be to copy the JavaDocs I made in the feature branch https://github.com/OpenHFT/Chronicle-Core/blob/feature/managedapi/src/main/java/net/openhft/chronicle/core/Jvm.java (if I remember correctly, only this class was documented)

Ignore Deprecations, moved classes and methods as we will address these issues in the future.
